sdk/daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala

```diff
@@ -45,12 +49,56 @@ class ExceptionTest(majorLanguageVersion: LanguageMajorVersion)
     SEApp(se, Array(SParty(p)))
   }
 
-  private val party = Party.assertFromString("Alice")
+  private val alice = Party.assertFromString("Alice")
+
+  private def runUpdateExpr(
+      compiledPackages: PureCompiledPackages,
+      expr: Expr,
+      getKey: PartialFunction[GlobalKeyWithMaintainers, Value.ContractId] = Map.empty,
+  ): Either[SError, SValue] = {
+    runUpdateExpr(
+      compiledPackages,
+      Map.empty,
+      compiledPackages.compiler.unsafeCompile(expr),
+      PartialFunction.empty,
+      getKey,
+    )
+  }
 
-  private def runUpdateExpr(pkgs1: PureCompiledPackages)(e: Expr): SResult[Question.Update] = {
```

Comment:
naive question, why did we change the output type ?

Reply:
Because I needed a version of `run` that accepts `getContract` and `getKey` and it has a different return type.

Reply:
fair enough.